src/common/commonutils/UserUtils.c

```diff
@@ -1495,7 +1515,7 @@ int CheckAllUsersHomeDirectoriesExist(char** reason, OsConfigLogHandle log)
     {
         for (i = 0; i < userListSize; i++)
         {
-            if (userList[i].noLogin || userList[i].cannotLogin || userList[i].isLocked)
+            if (userList[i].noLogin || userList[i].cannotLogin || userList[i].isLocked || userList[i].notInShadow)
```

Comment:
For this check and all of those below - those users should have a proper home directory .